addons/im_livechat/static/src/views/livechat_looking_for_help_controller_mixin.js

```diff
@@ -0,0 +1,48 @@
+import { LivechatViewControllerMixin } from "@im_livechat/views/livechat_view_controller_mixin";
+
+import { onWillDestroy } from "@odoo/owl";
+import { deserializeDateTime } from "@web/core/l10n/dates";
+
+import { useService } from "@web/core/utils/hooks";
+import { useDebounced } from "@web/core/utils/timing";
+
+const RELOAD_DEBOUNCE_DELAY = 300;
+
+export const LivechatLookingForHelpReloadMixin = (ViewController) =>
+    class extends LivechatViewControllerMixin(ViewController) {
+        setup() {
+            super.setup(...arguments);
+            const busService = useService("bus_service");
+            const reload = useDebounced(() => this.model.load(), RELOAD_DEBOUNCE_DELAY);
+            const onUpdate = ({ added_channel_ids, removed_channel_ids, dt }) => {
+                if (this.lastLoadDt > deserializeDateTime(dt)) {
```

Comment:
Won't this mix/match dates from client and server, which are possibly not exactly in sync?

Server value should normally only be compared against other server values, if anything at all.

Reply:
I'm not talking about timezone issue, but about clock issue.
If you compare client date with server date and the server is 100ms behind, and the bus notif takes less than that to arrive, it's possible a newer value will be ignored because the client thinks its in the past according to its clock.